crates/viewer/re_space_view_spatial/src/visualizers/videos.rs

```diff
@@ -199,34 +193,50 @@ impl VideoFrameReferenceVisualizer {
                         Some(texture)
                     }
                     FrameDecodingResult::Error(err) => {
-                        error_text = Some(err.to_string());
+                        self.show_video_error(
+                            render_ctx,
+                            spatial_ctx,
+                            world_from_entity,
+                            err.to_string(),
+                            video_size,
+                            entity_path,
+                        );
                         None
                     }
                 } {
-                    let rect =
-                        textured_rect_for_video_frame(world_from_entity, video_size, texture);
-                    rectangles.push(rect);
+                    let textured_rect = TexturedRect {
+                        top_left_corner_position: world_from_entity
+                            .transform_point3(glam::Vec3::ZERO),
+                        // Make sure to use the video instead of texture size here,
+                        // since it may be a placeholder which doesn't have the full size yet.
+                        extent_u: world_from_entity.transform_vector3(glam::Vec3::X * video_size.x),
+                        extent_v: world_from_entity.transform_vector3(glam::Vec3::Y * video_size.y),
```

Comment:
I think `video_resolution` is a better name than `video_size`.

At least I think of "resolution" as the immutable thing stored in the file, while "size" could be the display size, which depends on zoom etc.